app/routes/modern-web-podcast/index.js

```diff
@@ -1,15 +1,64 @@
 import Ember from 'ember';
 
-const ensureImage = (project) => (podcast) => project({
-  ...podcast,
+
+/*
+NOTES FOR TRACY
+=========================
+
+1. What is Object.assign? It's a way to copy an properties from a list of objects onto another object.
+   So if you do `Object.assign(a, b, c)` it's going to take all of the properties from b and c and copy them
+   onto `a`, then return `a`. If you want to duplicate an object, so you don't alter the original object, you'd
```

Comment:
An important note here is the overriding that happens

for instance:

`Object.assign({a: 1}, {a: 2}, {a: 3}) === {a: 3}`


another interesting thing is that everytime you do the copy approach is that you are generating a new object for every call which can end up generating a lot of garbage and result in a lot of time being spent in GC.